File: Websitedangtintimkiemnhatro/Websitedangtintimkiemnhatro/Models/Detail.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace Websitedangtintimkiemnhatro.Models
{
    public class Detail
    {
        [Key]
        public int Id { get; set; }
        public int NumberBath { get; set; }
        public int NumberLiving { get; set; }
        public Typeofnew Typeofnew { get; set; }
        [ForeignKey("TypeofnewId")]
        [Required]
        public int TypeofnewId { get; set; }
        public Motel Motel { get; set; }
    }
}

File: font/src/app/model/Detail.ts
import { Motel } from '../model/Motel';
import { Typeofnew } from '../model/Typeofnew';
import { LiveType } from '../model/LiveType';

export class Detail {
    id: Int32Array;
    numberBath: Number;
    numberLiving: Number;
    director: string;
    legal: string;
    directormain:string;
    liveTypeId: string;
    liveType: LiveType;
    typeofnewId: Int32Array;
    typeofnew: Typeofnew;
    motelid:Int32Array;
    motel:Motel;
}